pkgs/by-name/ya/yandex-music/ym_info.json

```diff
@@ -1,14 +1,6 @@
 {
-  "ym": {
-    "version": "5.18.2",
-    "exe_name": "Yandex_Music_x64_5.18.2.exe",
-    "exe_link": "https://music-desktop-application.s3.yandex.net/stable/Yandex_Music_x64_5.18.2.exe",
-    "exe_sha256": "a513eb6aa4bd05bae8503f5578a8e91d778794ffe4eb3a450040903570106941"
-  },
-  "electron": {
-    "version": "29.4.6",
-    "x64": "https://github.com/electron/electron/releases/download/v29.4.6/electron-v29.4.6-linux-x64.zip",
-    "armv7l": "https://github.com/electron/electron/releases/download/v29.4.6/electron-v29.4.6-linux-armv7l.zip",
-    "arm64": "https://github.com/electron/electron/releases/download/v29.4.6/electron-v29.4.6-linux-arm64.zip"
-  }
+  "version": "5.23.2",
+  "exe_name": "Yandex_Music_x64_5.23.2.exe",
+  "exe_link": "https://music-desktop-application.s3.yandex.net/stable/Yandex_Music_x64_5.23.2.exe",
+  "exe_sha256": "3e15e650657dfbdcdf459b9fc37155be93c6a952c0079d65af3eca268be6f82f"
```

Comment:
Use an SRI hash.

Reply:
I can not change this in original project. This file is auto-generated from [this](https://github.com/cucumber-sp/yandex-music-linux/blob/master/utility/version_info.json)

Reply:
Okay! I've googled a bit and found the way to convert. Add conversion to unpdateScript